### ISSUE TITLE
A11y: Close Menu in mobile Off-Canvas not keyboard accessible for certain pages

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#6997]
For pages like the ‘shipping costs’ or ‘privacy’ the off-canvas menu does not work with keyboard. Pressing enter does nothing and pressing space just makes it flicker. 

Additionally, closing with the mouse then redirects to the Home Page.

If there are other, similarly rendered pages, they are likely affected, too.

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40779, comment 540988]

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40779**



[created from NEXT-40779, comment 540990]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.3**

 This issue will be / was solved in Shopware version ***6.6.10.3***.



[created from NEXT-40779, comment 543025]
